sw/airborne/math/ActiveSetCtlAlloc/src/common/setupWLS.c

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (C) Till Blaha 2022-2023
+ * MAVLab -- Faculty of Aerospace Engineering -- Delft University of Techology
+ */
+
+/**
+ * @file setupWLS.c
+ * 
+ * @brief Implementations of setupWLS.h
+ */
+
+#include "setupWLS.h"
+#include <math.h>
+
+void setupWLS_A(
```

Comment:
Please format the code (and the rest of this lib). There is a script for that at the root of the project (`fix_code_style.sh`).